python/libcudf/libcudf/load.py

```diff
@@ -47,9 +47,14 @@ def _load_wheel_installation(soname: str):
 def load_library():
     """Dynamically load libcudf.so and its dependencies"""
     try:
-        # libkvikio must be loaded before libcudf because libcudf references its symbols
+        # librmm and libkvikio must be loaded before libcudf because libcudf references
+        # them. librmm does not yet have a load_library() function, so we just import
+        # it.
         import libkvikio
+        import librmm  # noqa: F401
```

Comment:
```suggestion
        # librmm and libkvikio must be loaded before libcudf because libcudf references
        # them.
        import libkvikio
        import librmm 
        librmm.load_library()
```

I know you'll remove the CI stuff in the next commit you push once there are `rmm` packages, but want to be sure this isn't missed... we now have `librmm.load_library()` in #1808 + still want `rapids_logger.load_library()` for include-what-you-use reasons.

This in-browser diff probably won't be exactly what `isort` / `ruff` want, might be easier to make this change manually.

Reply:
Thanks! Yeah I have this change already made locally, Bradley noticed the inconsistency as well.

Reply:
Fixed in https://github.com/rapidsai/cudf/pull/17899/commits/e5c5aa47c7d291f769fa7624c1d80ed9b782aa97.